backend/codesirius/internal_api/views/apikey.py

```
@@ -0,0 +1,53 @@
+        serializer = APIKeySerializer(data=request.data)
+        if serializer.is_valid(raise_exception=True):
+            api_key = serializer.save(created_by=request.user)
+            return CodesiriusAPIResponse(
+                data=APIKeySerializer(api_key).data,
+                status_code=status.HTTP_201_CREATED,
+                message="API key created",
+            )
+
+
+class APIKeyDestroyAPIView(APIView):
+    permission_classes = [IsStaff]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)